submissions/another96/html-css-popup/index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" href="style.css">
+      <title>My Popup</title>
+   </head>
+   <body>
+      <header>
+         <nav>
+            <ul class="vertical-nav">
+               <li tabindex="0" class="nav-item"><a href="#" class="link-focus" tabindex="-1"><img src="img/envelope.png" alt="messeges" class="icon-nav"></a></li>
+               <li tabindex="0" class="nav-item"><a href="#" class="link-focus" tabindex="-1"><img src="img/picture.png" alt="pictures" class="icon-nav"></a></li>
+               <li tabindex="0" class="nav-item">
+                  <input type="checkbox" id="popup-label" class="first-input">
+                  <label for="popup-label" class="label-focus">
+                  <img src="img/apps.png" alt="Open popup menu" class="icon-nav">
+                  </label>
+                  <div class="main-popup-container">
```

Comment:
Your popup is not accessible from the keyboard. The user must have the ability to open the popup with keyboard. [Watch video](https://share.vidyard.com/watch/LTjTDK9en1TAbH66XKxefN?)